static/src/javascripts/projects/common/modules/discussion/comment-box.js

```diff
@@ -215,8 +224,11 @@ class CommentBox extends Component {
     }
 
     postComment(): void {
+        // $FlowFixMe
+        const { value } = this.elem && this.elem.body;
```

Comment:
I guess Flow's unhappy here because if `this.elem` doesn't exist we'd get a `TypeError: Cannot match against 'undefined' or 'null'`? Should we refactor to be on the safe side?

Reply:
Not quite: https://github.com/guardian/frontend/pull/18037/commits/597738d2bbe9682c4371694758c79b0fb1e9db2f